### PR TITLE
[LA-278] - Fixed typos and content in README & aws deployment docs.

### PR DESCRIPTION
**Pull request by sandeepmjay**
Darlene found some typos in the READ.me and AWS deployment docs. Updated them for better readability.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,11 @@ node app
 
 ## Run
 
-Refresh the DataLake views. The views give fast access to enrollment and assignment data.
+Refresh the DataLake views. The materialized views are intended to give fast access to analytics data to the application layer.
```
Comment (johncrossman) [inline README.md]: @sandeepmjay The sentence above is not clear. How about, "These high-performance materialized views give applications access to analytics data."

File: README.md [hunk 2]
```diff
@@ -33,7 +33,11 @@ node app
...
 
-1. Configure the cron-like job. The `dataLake.cronJob.cronTime` config must have a [valid cron pattern](http://crontab.org).
+1. Configure the cron-like job and the script to run. The `dataLake.cron.tasks` configs must have a [valid cron pattern](http://crontab.org) and the cron script to run on trigger.
```
Comment (johncrossman) [inline README.md]: @sandeepmjay The above is also a bit unclear. :)  Are you describing two separate configurations? Or, one config for two processes?  Perhaps some sample code to illustrate?

File: README.md (hunk 1 above)
Comment (sandeepmjay) [reply README.md]: sounds good. Flipping it.

File: docs/aws-deployment.md [hunk 3]
```diff
@@ -1,11 +1,10 @@
 # Data Loch AWS deployment recipe
 
-Data Loch is an application that runs scheduled scripts to sync latest canvas data dumps (recovered from Canvas Data API) and refreshes cnavas table housed institutional data lake. The application uses AWS services like S3, Elastic Beanstalk and Redshift that are assembled together to create a highly available
-platform for running institutional analytics.
+Data Loch is an application that runs scheduled scripts to sync latest canvas data dumps (recovered from Canvas Data API) and refreshes Canvas tables housed in the institutional data lake. The application uses AWS services like S3, Elastic Beanstalk and Redshift Spectrum that are assembled together to create a highly available platform for running institutional analytics.
```
Comment (johncrossman) [inline docs/aws-deployment.md]: @sandeepmjay The text above reads like an executive summary. I recommend we move it to the top of README. One suggestion: describe Loch's purpose before describing implementation.

File: docs/aws-deployment.md [hunk 4]
```diff
@@ -42,9 +41,22 @@ platform for running institutional analytics.
...
   http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/eb-cli3-install.html
 
+## 3. Pre-deployment set up
+
+  ### Steps
+  1. Create dedicated S3 buckets.
+  2. Enable transfer acceleration.
+  3. Enable encryption using KMS keys.
+  4. Setup Lifecycle rules to retain only last 15 days of data in the the bucket/canvas-data/daily/*
+  5. Set up a single node Redshift cluster with firewalls
```
Comment (johncrossman) [inline docs/aws-deployment.md]: @sandeepmjay Nit-picky: "setup" vs. "set up".  We want the latter for the above.

File: docs/aws-deployment.md [hunk 5]
```diff
@@ -42,9 +41,22 @@ platform for running institutional analytics.
...
+  6. Create a Redshift spectrum role and provide necessary permissions to S3 bucket
```
Comment (johncrossman) [inline docs/aws-deployment.md]: Minor: capitalize 'spectrum'?

File: docs/aws-deployment.md [hunk 6]
```diff
@@ -1,11 +1,10 @@
...
 
-Data Loch is an application that runs scheduled scripts to sync latest canvas data dumps (recovered from Canvas Data API) and refreshes cnavas table housed institutional data lake. The application uses AWS services like S3, Elastic Beanstalk and Redshift that are assembled together to create a highly available
-platform for running institutional analytics.
+Data Loch is an application that runs scheduled scripts to sync latest canvas data dumps (recovered from Canvas Data API) and refreshes Canvas tables housed in the institutional data lake. The application uses AWS services like S3, Elastic Beanstalk and Redshift Spectrum that are assembled together to create a highly available platform for running institutional analytics.
 
-# Elastic Beanstalk deployment
+# AWS deployment recipe
 
-## 1. Pre-deployment set up
+## 1. AWS cli & eb cli set up
```
Comment (johncrossman) [inline docs/aws-deployment.md]: ...and in the above, you want 'setup' (noun)

**Comment by johncrossman** [comment]
@sandeepmjay - Thank you for the edits. I'll merge once amendments are in.

**Comment by sandeepmjay** [comment]
Thanks @johncrossman. Ship it !

**Pull request merged**